### PR TITLE
loading message added and invoice tab changed

### PR DESCRIPTION
**Pull request by Shruti-Apte**
## Notion card
NA
## Summary

- Added loading message 
- Shifted invoice tab to first place
## Preview

Screenshot 2023-01-02 at 6 21 01 PM

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor (non-breaking and retains same functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

- Tested working for loading message
- Invoice tab is shifted to first place
### Checklist:

- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [x] I have annotated changes in the PR that are relevant to the reviewer
- [ ] I have added automated tests for my code

### CODE REVIEW

File: app/javascript/src/components/Invoices/List/container.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ const Container = ({
   isDesktop,
 }) => {
   let appliedFilterCount = (filterParamsStr.match(/&/g) || []).length;
-
+  const [loading, setLoading] = useState<boolean>(false);
```
Comment (prasanthchaduvula) [inline app/javascript/src/components/Invoices/List/container.tsx:27]: Use `true` as default value for loading in the useState

File: app/javascript/src/components/Invoices/List/container.tsx [hunk 2]
```diff
@@ -48,7 +48,15 @@ const Container = ({
     });
   };
 
-  return invoices.length > 0 ? (
+  useEffect(() => setLoading(true), []);
```
Comment (prasanthchaduvula) [inline app/javascript/src/components/Invoices/List/container.tsx:51]: This not required any more after making this change 
`Use true as default value for loading in the useState`

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

**Comment by supriya3105** [comment]
@apoorv1316 @prasanthchaduvula Please test and review this PR today. @Shruti-Apte Please update the branch.

File: app/javascript/src/components/Invoices/List/container.tsx [hunk 3]
```diff
@@ -52,7 +52,15 @@ const Container = ({
     );
   };
 
-  return invoices.length > 0 ? (
+  useEffect(() => {
+    if (invoices.length > 0) setLoading(false);
```
Comment (prasanthchaduvula) [inline app/javascript/src/components/Invoices/List/container.tsx:56]: @Shruti-Apte What if invoice length is 0 then loading will be always true right? Instead its better to show no invoices found

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

**Comment by apoorv1316** [comment]
> @apoorv1316 @prasanthchaduvula Please test and review this PR today. @Shruti-Apte Please update the branch.

@supriya3105 Rpsec failing and page breaking because of a file name change. @Shruti-Apte is working on it. Will approve once it is fixed.

**Review by prasanthchaduvula — APPROVED** [review]

**Review by apoorv1316 — CHANGES_REQUESTED** [review]
@Shruti-Apte Please see loom https://www.loom.com/share/1d9b1ad557d844699223f9df85d9a064

**Review by apoorv1316 — APPROVED** [review]

**Pull request merged**